README.md

```diff
@@ -10,14 +12,37 @@
 <img alt="screenshot" src="./docs/images/screenshot.png" width="800px">
 </p>
 
-## One-line Installation
+## Installation
+
+### Binary
+
+Download the binary using onel-line command:
 
 ```console
 curl -fsSL https://raw.githubusercontent.com/greptimeteam/gtctl/develop/hack/install.sh | sh
 ```
 
 After downloading, the `gtctl` will be in the current directory.
 
+### Homebrew (macOS)
+
+On macOS, `gtctl` is available via Homebrew:
+
+```console
+brew tap greptimeteam/greptime
+brew install gtctl
+```
+
+### From Source
+
+If you already have the [Go](https://go.dev/doc/install) installed, you can run `make` command under this project to install `gtctl`:
+
+```console
+make gtctl
+```
+
+After building, the `gtctl` will be generated in `./bin/`.
```

Comment:
```suggestion
After building, the `gtctl` will be generated in `./bin/`. If you want to install `gtctl`, you can run the `install` command:

```console
# The built gtctl will be installed in /usr/local/bin.
make install

# The built gtctl will be installed in your customed path.
make install INSTALL_DIR=<your-path>
```
```